sockeye/inference.py

```diff
@@ -688,6 +710,8 @@ def make_input_from_multiple_strings(sentence_id: int, strings: List[str]) -> Tr
     ('score', float)
 ])
 
+EMPTY_TRANSLATION = Translation(target_ids=[], attention_matrix=np.asarray([[0]]), score=-np.inf)
```

Comment:
Wouldn't it be better to create a new object (similar to _bad_input() above). It doesn't happen at the moment AFAIK, but if at some point the returned object is modified we can get unexpected results (objects are passed by reference).

Reply:
Probably better yes, will change.